META.json

```diff
@@ -58,10 +58,11 @@
          "web" : "https://github.com/regentmarkets/duo_perl"
       }
    },
-   "version" : "0.01",
+   "version" : "0.02",
    "x_contributors" : [
       "Ben Murray <ben@duosecurity.com>",
       "Fayland Lam <fayland@gmail.com>",
+      "Raunak Kathuria <raunakkathuria@gmail.com",
       "Jon Oberheide <jon@oberheide.org>",
       "alichator <alishahc@umich.edu>"
    ]
```

Comment:
@arcofdescent But for this repo we have it in main repo, should this be there in main repo?